mesos-swarm-marathon/azuredeploy.json

```diff
@@ -153,9 +161,26 @@
     "chronosEnabled": {
       "type": "string",
```

Comment:
there is a bug in ARM where concat does not support bool.  You can repro this using this template:  https://github.com/anhowe/scratch/tree/master/bool.  ARM will fix in next release.  In the meantime, we have to use string.